packages/p2p/src/components/app.jsx

```diff
@@ -19,6 +20,25 @@ import MyProfile from './my-profile/my-profile.jsx';
 import './app.scss';
 
 const allowed_currency = 'USD';
+
+const TemporaryBarredMessage = () => {
+    const { general_store } = useStores();
+
+    const formatted_date = getFormattedDateString(new Date(convertToMillis(general_store?.user_blocked_until)));
```

Comment:
Could be `computed` value.

Reply:
You're right, it's better as a computed value.